tests/test_valdemarsro.py

```diff
@@ -0,0 +1,81 @@
+from recipe_scrapers.valdemarsro import Valdemarsro
+from tests import ScraperTest
+
+
+class TestValdemarsroScraper(ScraperTest):
+
+    scraper_class = Valdemarsro
+
+    def test_host(self):
+        self.assertEqual("valdemarsro.dk", self.harvester_class.host())
+
+    def test_title(self):
+        self.assertEqual(
+            "Madpandekager",
+            self.harvester_class.title(),
+        )
+
+    def test_category(self):
+        self.assertEqual("Bålmad,Familiefavoritter,Madpandekager og madvafler,Nem Hverdagsmad,Opskrifter,Opskrifter til børn - Hverdagsfavoritter børn,Pandekager,Tilbehør Aftensmad,Vegetar", self.harvester_class.category())
+
+    def test_total_time(self):
+        self.assertEqual(40, self.harvester_class.total_time())
+
+    def test_cook_time(self):
+        self.assertEqual(20, self.harvester_class.cook_time())
+
+    def test_yields(self):
+        self.assertEqual("10 stk.", self.harvester_class.yields())
+
+    def test_image(self):
+        self.assertEqual(
+            "https://www.valdemarsro.dk/wp-content/2020/10/madpandekager.jpg",
+            self.harvester_class.image(),
+        )
+
+    def test_ingredients(self):
+        self.assertEqual(
+            [
+                '4 æg',
+                '4 dl mælk',
+                '1 nip salt',
+                '125 g hvedemel',
+                '50 g grahamsmel',
+                '1 spsk smør, til stegning'
+            ],
+            self.harvester_class.ingredients(),
+        )
+
+    def test_instructions(self):
+        self.assertEqual(
+            """Pisk æg og mælk sammen med salt, grahamsmel og hvedemel.
+
+Smelt lidt smør eller kom lidt olie på en pande og steg 10 madpandekager en ad gangen, på begge sider, til de er flotte lysebrune.""",
+            self.harvester_class.instructions(),
+        )
+
+    def test_author(self):
+        self.assertEqual("Ann-Christine Hellerup Brandt", self.harvester_class.author())
+
+    def test_description(self):
+        self.assertEqual("""Klassiske madpandekager er nemme og lækre til at fylde med alverdens gode sager og kan nydes til både frokost og aftensmad.
+
+Man kan både fylde madpandekager ved bordet ala wraps eller man kan fylde dem med gode sager, rulle dem sammen og komme dem i et fad. Fordel lidt god friskrevet ost over og gratiner dem i ovnen. De er skønne at spise med en salat og dip til.
+
+Jeg deler en masse ekstra tips til min opskrift på madpandekager under opskriften
+
+----------
```

Comment:
Thanks for mentioning the dashed section in your introduction.  It could be a philosophical question, but I'm not certain whether we should include the divider and then the 'tips' items here.

I totally agree they're valuable -- and you're correct, there isn't (currently?) anywhere else for us to put them currently -- I just reckon that semantically, it may make sense for the `instructions` method to only return content that represents the steps involved in making the recipe.

Reply:
I did consider putting the tips in the `instructions`, but since that's an array of steps, I also came to the conclusion that it didn't make sense.

I don't feel strongly about including them at all, to be honest I only noticed that they exist while implementing this scraper, even after using the recipies from the site for years. I'll remove them with the possibility of a comeback if a `notes` feature is eventually added to this project.

Reply:
Thank you - I'm not certain whether my suggestion is the best way to go, but I'm OK to be somewhat responsible for that either way :)

I've filed #495 to track the addition of a `notes` feature; you're welcome to subscribe to that if you'd like to track along with progress (no commitment required, though).

Reply:
I think it's a decent approach to avoid fitting it into the wrong box.

Awesome that you've created an issue for including notes - I'll be keeping an eye on it!